packages/rest-client/src/api/file/getFiles.ts

```diff
@@ -0,0 +1,59 @@
+import { apiRequest, ApiRequestSettings } from '../../apiRequest'
+import { ServerErrorResponse } from '../../ServerErrorResponse'
+import { RestClientError } from '../../tools/RestClientError'
+
+export type GetFilesOrdering = 'datetime_uploaded' | '-datetime_uploaded'
+
+export type GetFilesOptions = {
+  from?: Date
+  removed?: boolean
+  stored?: boolean
+  limit?: number
+  ordering?: GetFilesOrdering
+}
+
+export type PaginatedResponse<R, T> = {
+  next: string
+  previous: string
+  total: number
+  perPage: number
+  results: R[]
+  totals: T
+}
+
+export type FileInfo = unknown
```

Comment:
This file will reworked in the future PRs. You can skip it's review.

Reply:
ok